.brv/context-tree/development/ito_workflow/coordination_branch_git_behavior.abstract.md

```
@@ -0,0 +1 @@
+Coordination branch git behavior initializes missing remotes with an empty-tree root commit, rejects empty git output, validates errors and branch names, and reserves changes in a detached temporary worktree.
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)